src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -72,6 +72,11 @@ export const VideoView: React.FC = () => {
   }, [video?.duration, timestampFromQuery])
 
   const channelId = video?.channel.id
+  const { videos } = useVideos(
+    { where: { channelId_eq: channelId, isPublic_eq: true, isCensored_eq: false } },
+    { skip: !channelId }
+  )
+  const shouldShowMoreVideos = videos && videos.length > 1
```

Comment:
Do we need to fetch data manually here? I think `InfiniteVideoGrid` can handle it on it's own if pass it the title?